<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataXMLDialog</class>
 <widget class="QDialog" name="MetadataXMLDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>extract metadata XML options</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Extract</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="Citation_checkBox">
        <property name="text">
         <string>Citation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="Authors_checkBox">
        <property name="text">
         <string>Authors</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="References_checkBox">
        <property name="text">
         <string>References</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="Projects_checkBox">
        <property name="text">
         <string>Projects</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="Events_checkBox">
        <property name="text">
         <string>Events</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="Parameters_checkBox">
        <property name="text">
         <string>Parameters</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="Coverage_checkBox">
        <property name="text">
         <string>Coverage</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="Keywords_checkBox">
        <property name="text">
         <string>Keywords</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="TechnicalInfo_checkBox">
        <property name="text">
         <string>Technical info</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ClearAll_pushButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Citation_checkBox</tabstop>
  <tabstop>Authors_checkBox</tabstop>
  <tabstop>References_checkBox</tabstop>
  <tabstop>Projects_checkBox</tabstop>
  <tabstop>Events_checkBox</tabstop>
  <tabstop>Parameters_checkBox</tabstop>
  <tabstop>Coverage_checkBox</tabstop>
  <tabstop>Keywords_checkBox</tabstop>
  <tabstop>TechnicalInfo_checkBox</tabstop>
  <tabstop>ClearAll_pushButton</tabstop>
  <tabstop>OK_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
